domino/core/pom.xml

```diff
@@ -20,6 +20,12 @@
 				<extensions>true</extensions>
 				<configuration>
 					<singleton>true</singleton>
+					<manifestLocation>META-INF</manifestLocation>
```

Comment:
That seems to be the case, but they're still created in the updateSite's jars. Is that because those jars are created differently? Does the location outside of /target/ cause a problem?